src/main/daml/Daml/Finance/Instrument/Swap/InterestRate/Instrument.daml

```diff
@@ -77,7 +77,7 @@ template Instrument
       getClaims Claim.GetClaims{actor} = do
         -- get the initial claims tree (as of the swap's acquisition time)
         let getCalendars = getHolidayCalendars actor calendarDataProvider
-        schedule <- rollSchedule getCalendars periodicSchedule holidayCalendarIds
+        (schedule, calendars) <- rollSchedule getCalendars periodicSchedule holidayCalendarIds
```

Comment:
```suggestion
        (schedule, _) <- rollSchedule getCalendars periodicSchedule holidayCalendarIds
```

as it seems the calendars are not used anywhere. Same for the other similar cases